frontend/cypress/integration/talentsearch/redirects.spec.js

```diff
@@ -0,0 +1,31 @@
+import { aliasQuery } from "../../support/graphql-test-utils";
+
+describe("Redirects", () => {
+
+
+
+  const expectToBeOnProfile = () => {
+    cy.wait("@gqlgetMeQuery");
+    cy.url()
+    .should('match', /users\/[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}\/profile$/g);
```

Comment:
Might be nice to have the regex for a UUID a named constant somewhere to make it clear what this insanity represents and allow it to be reused. :grin:

Reply:
Oh, I wanted to do that but i don't know how to do that with regex! 😓

Reply:
Thanks! Added in [move uuid regex to variable for more clarity](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4460/commits/1bd8aed6a5a3f93f5988c742c6361b15f42b07f2)